src/archivematicaCommon/lib/fileOperations.py

```diff
@@ -122,16 +122,10 @@ def writeToFile(output, fileName, writeWhite=False):
         print("No output, or file specified")
     return 0
 
-<<<<<<< HEAD
+
```

Comment:
Ops, that was a conflict I accidentally left behind when I pulled from upstream this morning.